index.html

```diff
@@ -2297,22 +2300,22 @@ <h2>Contexts and Vocabularies</h2>
         </p>
 
         <p>
-Implementations that perform RDF processing MUST treat the following
-JSON-LD vocabulary URL as already resolved, where the resolved document matches
-the corresponding hash values below.
+Implementations that perform RDF processing MUST treat
+the JSON-LD serialization of the vocabulary URL as already resolved, where the resolved document matches
```

Comment:
I think `resolved` here should be `dereferenced`, which *includes* resolving as well as retrieving the target document, which retrieval is the part that's really expected to be not done at runtime.
```suggestion
the JSON-LD serialization of the vocabulary URL as already dereferenced, where the dereferenced document matches
```